app/controllers/users_controller.rb

```diff
@@ -0,0 +1,51 @@
+
+class UsersController < ApplicationController
+  # app/controllers/users_controller.rb
+  def index 
+    @users = User.all
+  end
+
+
+  def show
+    user_id = params[:id]
```

Comment:
Finding a logged in user is good functionality to put in a controller filter because it may be used multiple places in the future.